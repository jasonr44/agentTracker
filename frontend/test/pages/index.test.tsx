import React from 'react';
import { render } from '../testUtils';
import App from 'pages/index';

describe('App', () => {
  it('Matches snapshot', () => {
    const { asFragment } = render(<App />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
