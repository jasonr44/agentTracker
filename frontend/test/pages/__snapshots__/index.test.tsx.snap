// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-disableGutters"
  >
    <div
      class="makeStyles-root-1"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <button
            aria-label="menu"
            class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2 MuiIconButton-colorInherit MuiIconButton-edgeStart"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <h6
            class="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
          >
            Home
          </h6>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Login
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </header>
    </div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-4 MuiPaper-elevation1 MuiPaper-rounded"
      data-testid="card-component"
    >
      <button
        class="MuiButtonBase-root MuiCardActionArea-root"
        tabindex="0"
        type="button"
      >
        <img
          alt="Valorant Content"
          class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img"
          height="140"
          src="/public/images/gunBundle.jpg"
          title="Valorant Content"
        />
        <div
          class="MuiCardContent-root"
        >
          <h2
            class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
          >
            Valorant Content
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
          >
            See the current in game content.
          </p>
        </div>
        <span
          class="MuiCardActionArea-focusHighlight"
        />
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <div
        class="MuiCardActions-root MuiCardActions-spacing"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
          name="Share"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Share
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
